app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -241,7 +241,7 @@ class NavigationDrawerActivityTest {
           withId(R.id.profile_progress_text_view),
           isDescendantOfA(withId(R.id.header_linear_layout))
         )
-      ).check(matches(withText("1 Story Completed | 1 Topic in Progress")))
+      ).check(matches(withText("1 Story Completed | 0 Topics in Progress")))
```

Comment:
Add one test in this file where there is atleast `1 Topic in Progress`.

You can take help from `storyProfileTestHelper`. Also you might have to rename `testNavDrawer_openNavDrawer_profileProgressIsDisplayed ` accordingly.

Reply:
I have modified this test so that there is `1 Topic in Progress`. Keeping both the tests: 
1. `1 Story Completed | 0 Topics in Progress` 
2. `1 Story Completed | 1 Topic in Progress` 

is resulting in the failure of the first test, because the second test runs before the first test, which leads to the failure of the first test.  Hence, instead of adding another test, I have updated the original test so that there is at least `1 Topic in Progress`